autoload/neomake/makers/ft/rst.vim

```diff
@@ -23,3 +23,19 @@ function! neomake#makers#ft#rst#rstcheck()
             \ '%E%f:%l: (SEVERE/4) %m',
         \ }
 endfunction
+
+" TODO: determine proper path of current .rst file upwards in sear for conf.py file, use that instead of "docs"
```

Comment:
For this we want to find the `conf.py`'s folder upwards from the current file, right?
`fnamemodify(findfile('conf.py', './;'), ':h')` might work for this.
It should be ensured that this temporarily unsets `d` in `'cpoptions'` then.
This could make sense to have in a `utils` function.